Signal/src/ViewControllers/ConversationView/ConversationInputToolbar.m

```diff
@@ -69,18 +71,25 @@ - (void)dealloc
     [self removeKVOObservers];
 }
 
+- (CGSize)intrinsicContentSize
+{
+    CGSize newSize = CGSizeMake(self.bounds.size.width, self.toolbarHeight + ConversationInputToolbarBorderViewHeight);
+    return newSize;
```

Comment:
It might be helpful to add a comment about why this is necessary.

Reply:
Will do.